app/src/main/java/com/xa/xpensauditor/TabFragment.java

```diff
@@ -131,6 +173,11 @@ public void onCancelled(FirebaseError firebaseError) {
         registerForContextMenu(recyclerView);
 
         mAdapter1.setOnItemClickListener(new TransAdapter.ClickListener() {
+            /**
+             * When a transaction is clicked, it shows the details of the transaction.
+             * @param position
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.NonEmptyAtclauseDescriptionCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>At-clause should have a non-empty description.